tilingManager/tilingLayouts/maximize.js

```diff
@@ -32,43 +34,74 @@ var MaximizeLayout = class MaximizeLayout extends BaseTilingLayout {
     }
 
     onTileRegulars(windows) {
-        if (this.animationInProgress) return;
```

Comment:
Why do we remove this?

Reply:
We might not want to skip a tile even if it's animating.

Reply:
I restored it but I don't think it's useful